gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/CopyConfiguration.java

```diff
@@ -51,6 +51,8 @@ public class CopyConfiguration {
   public static final String COPY_PREFIX = "gobblin.copy";
   public static final String PRESERVE_ATTRIBUTES_KEY = COPY_PREFIX + ".preserved.attributes";
   public static final String PRESERVE_MODTIME_FOR_DIR = COPY_PREFIX + ".preserve.mod.time.for.dir";
+  //Setting this to be false means we will not overwrite the owner and permission if dir already exist on dst
+  public static final String ALWAYS_OVERWRITE_DIR_OWNER_AND_PERMISSION = COPY_PREFIX + ".alwaysOverwriteDirOwnerAndPermission";
```

Comment:
two thoughts here:
1. isn't this manifest-file-specific?  for other finders, won't dirs already on dest be skipped, whereas only manifest-based copy can ensure 'always'?  if so, let's qualify the name accordingly.
2. "always overwrite" may not be quite accurate, since that would imply we always do work, even when nothing to change (the fact might be reflected in an updated mod time).  perhaps more accurately something like `resyncOwnerAndPermissionEvenIfDirExists`.